### DISCUSSION TITLE
Why I cannot have a durable at the beginning of the flow

### DISCUSSION THREAD

**Discussion by yatw** [postmanlabs/postman-flows#377] [General]
I want to set my access token manually once at the first durable block, and have this access token be accessible to all following request. What is the way to do this?
<img width="710" alt="Screen Shot 2022-08-18 at 22 34 59" src="https://user-images.githubusercontent.com/19565177/185550155-b2d8966b-29bd-47e5-af4c-ccd85d3da338.png">

**Comment by yatmanwong** [comment]
solved by V2
